contact.html

```diff
@@ -0,0 +1,170 @@
+                </div>
+                <div class="col">
+                    <b>Benyam Worku</b>
+                </div>
+            </div>
+            <div class="row">
+                <div class="col">
+                    Trainee Developer
+                </div>
+            </div>
+        </a>
+        <button class="navbar-toggler " type="button" data-toggle="collapse" data-target="#myNav"
+            aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse " id="myNav">
+            <div class="navbar-nav ml-auto  ">
+                <a class="nav-item nav-link active" href="index.html" style="color:blue">ABOUT ME <span
+                        class="sr-only">(current)</span></a>
+                <a class="nav-item nav-link" href="resume.html">RESUME</a>
+                <a class="nav-item nav-link" href="projects.html">PROJECTS</a>
+                <a class="nav-item nav-link" href="contact.html">CONTACT</a>
+            </div>
+        </div>
+    </nav>
+    <!-- </header> -->
+
+<!-- the form , may be remove contact-wrapper-->
+<div class="contact-wrapper">
+    <div class=" form-heading container d-flex justify-content-center align-items-center ">
+        <div class="row">
+            <div class="col-2">
+                <div style="color:blue ">&#9632;</div>
+            </div>
+            <div class="col">
+                <h1>Let's talk</h1>
+            </div>
+        </div>
+        
+    </div>
+    <!-- form with multiple entries -->
+    <div class="form-container container">
+        <div class="row">
+            <div class="col-12">
+                <form  action="" class="contact-form" >
+                    <div class="form-group">
+                        <label for="first name">First Name</label>
+                        <input type="text" name="first name" id="first name" class="form-control">
+                    </div>
+                    <div class="form-group">
+                        <label for="last name">Last Name</label>
+                        <input type="text" name="last name" id="last name" class="form-control">
+                    </div>
+                    <div class="form-group">
+                        <label for="email ">Email</label>
+                        <input type="email" name="email" id="email" class="form-control">
+                    </div>
+                    <div class="form-group">
+                        <label for="subject">Subject</label>
+                        <input type="text" name="subject" id="subject" class="form-control">
+                    </div>
+                    <div class="form-group">
+                        <label for="message">Message</label>
+                        <input type="text" name="message" id="message" class="form-control">
```

Comment:
Message text box is wider and bigger than the rest of the text boxes. So you should be using ``textarea`` instead